python/ray/worker.py

```diff
@@ -927,25 +928,81 @@ def print_to_stdstream(data):
     print_worker_logs(data, print_file)
 
 
+# Start time of this process, used for relative time logs.
+t0 = time.time()
+autoscaler_log_fyi_printed = False
+
+
+def filter_autoscaler_events(lines):
+    global autoscaler_log_fyi_printed
+
+    if not AUTOSCALER_EVENTS:
+        return
+
+    # Print out autoscaler events only, ignoring other messages.
+    for line in lines:
+        if ":event_summary:" in line:
+            if not autoscaler_log_fyi_printed:
+                yield ("Tip: use `ray status` to view detailed "
+                       "autoscaling status. To disable autoscaler event "
+                       "messages, you can set AUTOSCALER_EVENTS=0.")
+                autoscaler_log_fyi_printed = True
+            yield line.split(":event_summary:")[1]
+
+
+def time_string():
```

Comment:
type annotation

Reply:
Hey Eric, can you please use a timestamp? I feel like the relative time is not user friendly because I will have to keep checking when the autoscaler started running and relative to that to estimate when things happened. After a few hours of autoscaler running, it will be very hard to have a meaningful relative time.

Reply:
nack, relative time is easier for users to understand when running their script. You can still access the absolute timestamps from the raw autoscaler logs.

Note that this is relative from *start of the job*, not from start of the cluster. So relative time is meaningful even in the case of a long running cluster with many jobs.